2-backend-basics/2.0-module-2-overview.md

```diff
@@ -85,7 +85,7 @@ For Windows, Module 2 marks a point that we will change the way files are stored
 Get access to all the files in your WSL Ubuntu instance with this path:
 
 ```
-\\$wsl\Ubuntu
+\\wsl$\Ubuntu-20.04
```

Comment:
Can we remove the `\Ubuntu-20.04` from the path, worried that in the future the version might get bumped and the link will no longer work!